### PR TITLE
build: updates project name and references.

### PR DESCRIPTION
**Pull request by matthewhartstonge**
Adds the goodies

### CODE REVIEW

**Comment by matthewhartstonge** [comment]
Oop - need to migrate to Yarn.

File: README.md [hunk 1]
```diff
@@ -1,44 +1,20 @@
-ember-spreadsheet-export
-==============================================================================
+# @linc-technologies/ember-spreadsheet-export
 
 Addon that encapsulates ability to render a data set as either excel or csv.
 
-Forked from [roofstock/ember-cli-data-export](https://github.com/roofstock/ember-cli-data-export).
```
Comment (hero-david) [inline README.md:6]: Not keeping the original credit?

Comment (hero-david) [reply README.md:6]: Ah, shifted below.

**Review by hero-david — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (matthewhartstonge) [reply README.md:6]: Yeah, decided as a dev it's not the first thing you care about - you wanna know compat, install and usage upfront.

File: package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "ci": "yarn install --ignore-engines --frozen-lockfile",
```
Comment (hero-david) [inline package.json:22]: Is ignore engines still needed on node 14? And do we want to adjust the engines below?

**Review by hero-david — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (matthewhartstonge) [reply package.json:22]: For now I'm keeping it all similar across all projects, but will remove in future.
From memory there is something that causes an issue (requires newer node) ...

`ember-cli-update`/`ember` declare the engine preference, so I'm keeping it where ember _believes_ the engines should be :)

Cheers man!

**Pull request merged**